compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -795,6 +795,8 @@ public enum DiagnosticErrorCode implements DiagnosticCode {
             "BCE4042", "worker.send.action.not.allowed.in.lock.statement"),
     WORKER_RECEIVE_ACTION_NOT_ALLOWED_IN_LOCK_STATEMENT(
             "BCE4043", "worker.receive.action.not.allowed.in.lock.statement"),
+    UNSUPPORTED_EMPTY_CHARACTER_CLASS(
+            "BCS4041", "unsupported.empty.character.class")
```

Comment:
```suggestion
            "BCS4045", "unsupported.empty.character.class")
```

Please re-sync with the master to get the latest changes.